### PR TITLE
Improvement/add readme quickstart

### LINKED ISSUE #15: Add a "Quick Start" Section to ReadMe
Understandably, the readme for this box is quite dense. I think an additional section after requirements, labeled "Quick Start" with the main commands a user needs would be a useful addition.

### PR DESCRIPTION
**Pull request by dkillen**
The README for the StarkNet box provides a lot of information for a user to digest. This PR adds a section titled "Quick Start" toward the top of the README file and condenses the Installation and Setup sections into a single section titled "Installation and Setup".

Fixes #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,14 +44,74 @@ The StarkNet Box has the following requirements:
...
+```bash
+npm run starknet:invoke --contract=contract --address=<contract_address> --function=increase_balance 1000
+```
+
+### 7. Run unit tests
+
+```bash
+npm run starknet:test
+```
+
```
Comment (kevinweaver) [inline README.md:104]: I suggest we add `npm run starknet:start_devnet`  to this list as well

**Review by kevinweaver — APPROVED** [review]
One suggestion, but otherwise this is super helpful to surface the main commands at the top!

File: README.md [hunk 2]
```diff
@@ -42,14 +44,74 @@ The StarkNet Box has the following requirements:
 
 **Important Note:** The scripts included in this Box rely on Docker being available and running. On Linux you may need to have Docker configured to [run the Docker daemon in 'rootless' mode](https://docs.docker.com/engine/security/rootless/).
 
-## Installation
+## Quick Start
+
+The following commands demonstrate how to use the StatkNet box and will help you to get up and running quickly. For more details, you are encouraged to read this document in full.
```
Comment (fainashalts) [inline README.md:49]:
```suggestion
The following commands demonstrate how to use the StarkNet box and will help you to get up and running quickly. For more details, you are encouraged to read this document in full.
```

**Review by fainashalts — APPROVED** [review]
LGTM, except one typo! Thanks Dave!

File: README.md (hunk 1 above)
Comment (dkillen) [reply README.md:104]: Good catch. I agree this should also be in the Quickstart.

**Pull request merged**